geth/transactions/txqueue_manager_test.go

```diff
@@ -90,16 +91,25 @@ func (s *TxQueueTestSuite) TestCompleteTransaction() {
 	s.setupTransactionPoolAPI(account, nonce, gas, nil)
 
 	txQueueManager := NewManager(s.nodeManagerMock, s.accountManagerMock)
-
+	if disableNotifications {
```

Comment:
they are always disabled in unit tests, aren't they?

Reply:
It was not disabled in `TestCompleteTransaction` so I left it out as an option. I can remove it if that sounds better.